global/scripts/MyelinMap_BC.sh

```diff
@@ -94,19 +94,38 @@ log_Msg "RegNameInOutputName: $RegNameInOutputName"
 log_Msg "RegNameInT1wName: $RegNameInT1wName"
 log_Msg "RegNameStructString: $RegNameStructString"
 
+# check file exists
 NativeMyelinMap=${NativeFolder}/${Subject}.MyelinMap.native.dscalar.nii
-log_File_Must_Exist "${NativeMyelinMap}"
+log_File_Must_Exist "${NativeMyelinMap}" # 1
+LowResCiftiTemplate=${LowResFolder}/${Subject}.MyelinMap.${LowResMeshString}.dscalar.nii
+log_File_Must_Exist "${LowResCiftiTemplate}" # 2 
+LeftSphereCurrentSphere=${NativeFolder}/${Subject}.L.sphere${RegNameStructString}.native.surf.gii
+log_File_Must_Exist "${LeftSphereCurrentSphere}" # 3
+LeftSphereNewSphere=${LowResFolder}/${Subject}.L.sphere.${LowResMeshString}.surf.gii
+log_File_Must_Exist "${LeftSphereNewSphere}" # 4
+LeftAreaSurfCurrentArea="${NativeT1wFolder}/${Subject}.L.midthickness.native.surf.gii"
+log_File_Must_Exist "${LeftAreaSurfCurrentArea}" # 5
+LeftAreaSurfNewArea=${LowResFolder}/${Subject}.L.midthickness${RegNameInOutputName}.${LowResMeshString}.surf.gii
+log_File_Must_Exist "${LeftAreaSurfNewArea}" # 6 - This is the one that doesn't exist
```

Comment:
The "doesn't exist" part of the comment is ancient cruft that should be deleted.

Reply:
Also, I don't think the numbers serve any purpose, so I would just delete the comments from all of these.